share/data-modeling/domain-driven-design/pattern1/README.md

```diff
@@ -0,0 +1,183 @@
+# Pattern 1: Single Domain, Single Bounded Context, and a Single Uni
+
+
+# Scenario
+
+The customer Relationship Management(CRM) Team wants to build a single source of truth for customer account data which includes first name, last name, customerId, personal details, contact details, and address. The CRM team will frequently query customer data using customerId. 
+
+
+# Modeling
+
+Since CRM is an independent function we will assume it is a domain on its own. Inside CRM, we will have multiple sub-domains; customer account is one such sub-domain. You can model a customer account as a Bounded Context.  Since CRM will perform frequent lookup, search, and filter on customer records by customerId, you will index customer accounts on customerId property. 
```

Comment:
```suggestion
Since CRM is an independent business unit we will assume it is a domain on its own. Inside CRM, we will have multiple sub-domains; customer account is one such sub-domain. You can model a customer account as a Bounded Context.  Since CRM will perform frequent lookup, search, and filter on customer records by customerId, you will index customer accounts on customerId property. 
```